index.browser.json

```diff
@@ -17,6 +15,7 @@
         "./filter-button",
         "./filter-menu-button",
         "./filter-menu",
+        "./global",
```

Comment:
Sorry, just notice this. Can we move global and utility back to their original place? I believe they need to be first in the cascade. Possibly not an issue, but I'd rather not risk any regressions until I've had a chance to be sure.

Reply:
Duplicate comment. Closing